src/pwr.ps1

```diff
@@ -97,7 +97,7 @@ param (
 	[string[]]$Run
 )
 
-$WriteToHost = ($MyInvocation.ScriptLineNumber -eq 1) -and ($MyInvocation.OffsetInLine -eq 1) -and ($MyInvocation.PipelineLength -eq 1)
+$WriteToHost = ($MyInvocation.ScriptLineNumber -eq 1) -and ($MyInvocation.PipelineLength -eq 1)
```

Comment:
What was the original intent here? Was it to just use normal output when the command is part of a pipeline, so the output can be processed using `stdout` (fd 1) and `stderr` (fd 2), instead of the `Information Stream` (fd 6)?

I'm thinking we need to re-evaluate when this happens. I like using `Write-Host` for the prompt, but I'm not sure about any other situation. Maybe for use with `-Run` and/or when running in `shell` so the `pwr` output is separate from the command output? If so, then I'm thinking something like

```suggestion
$WriteToHost = $Run -or $env:PwrShellPackages
```

But, that may seem a little funky when starting or exiting a shell. @tbateson, @troyaf, thoughts?